README.md

```diff
@@ -70,6 +70,33 @@ After installing these dependencies, please run this command to install the code
 python setup.py develop
 ```
 
+We also provide a [Dockerfile](docker/Dockerfile) to ease environment setup. To get started with docker, please make sure that `nvidia-docker` is installed on your machine. After that, please execute the following command to build the docker image:
+
+```bash
+cd docker && docker build .
```

Comment:
Use `docker build -t` to assign a tag to the docker.

Reply:
Fixed.